src/users/userSelectors.js

```diff
@@ -188,6 +196,40 @@ export const getAccountDetailsUserForEmail: Selector<UserOrBot, string> = create
   },
 );
 
+/**
+ * The user with the given user Id.
+ *
+ * In particular, returns the user for:
+ * * i) cross-realm bots
+ * * ii) deactivated users (`is_active` false; see `User` and the linked docs)
```

Comment:
I want to word this part a bit differently; I'll do that as an added commit on top.

Two concrete things I notice:
* "returns the user for cross-realm bots [and] deactivated users" is kind of confusing; it sounds like other users (the common case) aren't included, or maybe like it returns a user that could be called "the user *for*" a cross-realm bot.
* "See also" is a great technique to provide more context :slightly_smiling_face: See e.g. some mentions in the Flutter style guide (which in general is great, and especially great about docs style), [here](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#leave-breadcrumbs-in-the-comments) and [here](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#writing-prompts-for-good-documentation).

Reply:
I saw https://github.com/zulip/zulip-mobile/commit/a01decc482e41489639dfa903f3089fceb7657cf, it's more clear now.

This may be off-topic, but how did you end up reading the Flutter style guide in such depth?

Reply:
> how did you end up reading the Flutter style guide in such depth?

A couple of years ago when I first started doing mobile development, Flutter had just entered beta and looked pretty cool, so I spent some time working with it. One of the things I was impressed by was the quality of the docs, and relatedly of the code and APIs. Here's a past chat thread with a few thoughts on Flutter: [thread](https://chat.zulip.org/#narrow/stream/2-general/topic/Will.20the.20Zulip.20devs.20consider.20moving.20to.20Flutter.3F/near/802275).